src/components/Footer/FooterMenuSection.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classnames from 'classnames';
+
+interface IMenuItem {
+  label: string;
+  href: string;
+}
+
+interface Props {
+  title: string;
+  items: IMenuItem[];
+  className?: string;
+}
+
+const FooterMenuSection: React.FC<Props> = ({ title, items, className }) => {
```

Comment:
I will avoid to have a prefix of `Footer` here as you are already within the Footer folder